src/rgw/setup/rgw_service.py

```diff
@@ -15,20 +15,28 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
 import os
+import sys
+import shlex
 from cortx.utils.log import Log
 from cortx.rgw.setup.error import SetupError
 from cortx.utils.conf_store import MappedConf
 
 
-class RgwStart:
+class RgwService:
     """Entrypoint class for RGW."""
 
     @staticmethod
-    def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
+    def start(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
+            cmd = shlex.split(cmd)
```

Comment:
duplicate variable name 'cmd'. Either change above line or below

Reply:
I am reusing this variable.

Reply:
the purpose of line 32 & 33 are different. Line 32 stores string and line 33 stores a list.
Even though python is flexible to overwrite any variables, lets follow some good standards by keeping a meaningful variable name.

Same variable name can be reused in case of overwriting with same datatype.
For ex: 
```
1. bool a = True
    bool a = False
2. for i in (0,5):
        i = i+1
```

Reply:
cmd contains the command to be run in both the lines.
This is a very common practice AFAIK, to deduce what variable contains.

and variable name is 'cmd', which is meaningful enough in this case.

I was not aware of any such coding standard.. It this from PEP8 or something?